js/src/Figure.ts

```diff
@@ -922,13 +929,23 @@ export class Figure extends widgets.DOMWidgetView {
     }
 
     _update_gl() {
-        this.render_gl()
         this._update_requested = false;
+        this.render_gl();
```

Comment:
should this maybe change to?
```suggestion
        await this.render_gl();
        this._update_requested = false;
```

Reply:
`this._update_requested = false;` must be before the `render_gl` call, if we want to fix the issue